scripts/move_wcs.py

```diff
@@ -96,14 +96,14 @@ def add_default_keywords(new_hdr):
     wcsaxes = new_hdr['WCSAXES']
     if wcsaxes == 3:
         default_pc = {"PC1_1": 1, "PC1_2": 0, "PC1_3": 0,
-                      "PC2_1": 0 , "PC2_2": 1, "PC2_3": 0,
+                      "PC2_1": 0, "PC2_2": 1, "PC2_3": 0,
```

Comment:
We don't have data with 3 wcsaxes any more. This script should be revised, in another PR.